llvm/lib/CodeGen/GlobalISel/LegalityPredicates.cpp

```diff
@@ -35,7 +35,7 @@ LegalityPredicates::typeInSet(unsigned TypeIdx,
                               std::initializer_list<LLT> TypesInit) {
   SmallVector<LLT, 4> Types = TypesInit;
   return [=](const LegalityQuery &Query) {
-    return llvm::is_contained(Types, Query.Types[TypeIdx]);
+    return llvm::is_contained(std::move(Types), Query.Types[TypeIdx]);
```

Comment:
This seems really weird. is_contained is a template function that takes a universal reference. Why do we need a move here?

Reply:
Sorry, I missed that. I've removed these.